CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 ## 1.4.0 (Unreleased)
+
```

Comment:
Let's set the version number to 1.3.2 for this since all of the changes are under the hood.